media-api/test/lib/elasticsearch/impls/elasticsearch1/MediaApiElasticSearch1Test.scala

```diff
@@ -44,7 +46,30 @@ class MediaApiElasticSearch1Test extends ElasticSearchTestBase with Eventually w
 
   private val expectedNumberOfImages = images.size
 
+  def findElasticsearchConfig(): Path = {
+    // SBT does some monkey stuff and runs the test from the "thrall" folder compared to IntelliJ that runs it from the root
+    val sbtsIdeaOfCwd = Paths.get("").toAbsolutePath
+
+    val cwd = if (sbtsIdeaOfCwd.endsWith("media-api")) {
+      sbtsIdeaOfCwd.getParent
+    } else {
+      sbtsIdeaOfCwd
+    }
+
+    cwd.resolve("elasticsearch/test/elasticsearch.yml")
+  }
+
+  def esContainer = Some(DockerContainer("elasticsearch:1.7.1")
```

Comment:
I'm not super keen on the removal of the docker-compose files in favour of this. But happy to see how it turns out.